.github/workflows/ci.yml

```diff
@@ -66,6 +66,8 @@ jobs:
       - name: Install Dependencies
         run: |
           python -m pip install -r dev-requirements.txt
+      - name: 'Setup: OpenSearch environment'
+        run: echo "OPENSEARCH_URL=http://localhost:9200" >> $GITHUB_ENV
```

Comment:
Where is this variable used? Also, what happens for test clusters with security?